<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-9EJA6NL" ignored="0" name="Test" tests="5" failures="3" timestamp="09 Aug 2019 06:22:24 GMT" time="62.408" errors="0">
  <testcase name="add01_addSimpleTask" time="1.747" classname="tests.AddingTaskTest">
    <failure type="java.lang.AssertionError" message="expected [1] but found [25]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [25]
at tests.AddingTaskTest.add01_addSimpleTask(AddingTaskTest.java:22)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- add01_addSimpleTask -->
  <testcase name="add02_addMoreTasks" time="1.039" classname="tests.AddingTaskTest">
    <failure type="java.lang.NumberFormatException" message="For input string: &amp;quot;&amp;quot;">
      <![CDATA[java.lang.NumberFormatException: For input string: ""
at java.lang.NumberFormatException.forInputString(Unknown Source)
at java.lang.Integer.parseInt(Unknown Source)
at java.lang.Integer.parseInt(Unknown Source)
at pageobjects.TasksPage.getNumOfTasks(TasksPage.java:67)
at tests.AddingTaskTest.add02_addMoreTasks(AddingTaskTest.java:36)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- add02_addMoreTasks -->
  <testcase name="search" time="1.344" classname="tests.SearchTest"/>
  <testcase name="new01_createListWith" time="5.399" classname="tests.NewListTest">
    <failure type="java.lang.AssertionError" message="expected [I expected this] but found [I got this]">
      <![CDATA[java.lang.AssertionError: expected [I expected this] but found [I got this]
at tests.NewListTest.new01_createListWith(NewListTest.java:21)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- new01_createListWith -->
  <testcase name="new02_newList1newList2" time="6.479" classname="tests.NewListTest"/>
</testsuite> <!-- Test -->
